src/TitleDetails/hooks/useHoldingItems/useHoldingItems.js

```diff
@@ -0,0 +1,40 @@
+import { useQuery } from 'react-query';
+
+import {
+  useOkapiKy,
+  useNamespace,
+} from '@folio/stripes/core';
+import {
+  LIMIT_MAX,
+  ITEMS_API,
+} from '@folio/stripes-acq-components';
+
+export const useHoldingItems = (
+  holdingsRecordId,
+  kyOptions = {},
+  queryOptions = {},
+) => {
+  const ky = useOkapiKy();
+  const [namespace] = useNamespace();
+
+  const queryKey = [namespace, 'items', holdingsRecordId];
+
+  const options = {
+    ...kyOptions,
+    searchParams: {
+      limit: LIMIT_MAX,
+      ...kyOptions?.searchParams,
+      query: `holdingsRecordId==${holdingsRecordId}`,
+    },
+  };
+
+  const queryFn = () => ky.get(ITEMS_API, options).json();
+  const { data, isLoading, isFetching } = useQuery({ queryKey, queryFn, ...queryOptions });
```

Comment:
plz disable query when holdingsRecordId is not defined, no reason to make a request when no holding connected

Reply:
updated